contributors/design-proposals/cpu-manager.md

```diff
@@ -329,6 +331,16 @@ func (p *dynamicPolicy) UnregisterContainer(s State, containerID string) error {
    directly from the shared pool, is too simplistic.
     1. Mitigation: defer supporting this until a new policy tailored for
        use with `isolcpus` can be added.
+1. CPU exhaustion. If all available CPUs are allocated exclusively,
+   additional steps must be taken to remove any best-effort and
+   zero-CPU burstable pods from the node. In addition, the system must
```

Comment:
might just reduce

> best-effort and zero-CPU burstable pods

to

> pods with no CPU request, referred to here as 'no-CPU pods'"

Then you can use the no-CPU term in place of zero-CPU.  no-CPU is more accurate in my mind.